### PR TITLE
Implemented actuate method in Actuator

### PR DESCRIPTION
**Pull request by hexfati**

### CODE REVIEW

File: src/domain/feeder_actuator.py [hunk 1]
```diff
@@ -30,6 +30,10 @@ def send_command(self, cmd):
             )
             raise Exception("Unknown command")
 
+    def actuate(self):
+        """Method to trigger the FeederActuator sending it the OPEN Command"""
+        self.send_command(self.Commands.OPEN)
```
Comment (stefanodellosa-personal) [inline src/domain/feeder_actuator.py:35]: should we rather close the feeder whenever a detection occours? perhaps i didn't understand the command or the lable is misleading...

File: src/domain/feeder_actuator.py [hunk 2]
```diff
@@ -30,6 +31,10 @@ def send_command(self, cmd):
             )
             raise Exception("Unknown command")
 
+    def actuate(self):
+        """Method to trigger the FeederActuator sending it the OPEN Command"""
```
Comment (stefanodellosa-personal) [inline src/domain/feeder_actuator.py:35]: update method description to align with actuall command: OPEN -> CLOSE

**Review by stefanodellosa-personal — APPROVED** [review]

**Comment by alessandropalla** [comment]
Funny enough, the test_panorama photo changed and now there is a cow in it! :D and the test correctly identified it an fails.

**Review by alessandropalla — APPROVED** [review]

**Review by cdimauro — COMMENTED** [review]
LGTM

**Pull request merged**